src/App.vue

```diff
@@ -15,10 +15,18 @@
  */
 import LoadingProgress from '@/topbar/components/LoadingProgress'
 
+import { useClearDataOnLogout } from '@/utils/composables'
+import { useOffersUpdater } from '@/offers/queries'
+
 export default {
   components: {
     LoadingProgress,
   },
+  setup () {
+    // Global kind of things can be registered here
+    useOffersUpdater()
```

Comment:
Essentially, we might always be watching if any websocket update invalidates any query cache... that's a lot of listening, but probably just fine.

Reply:
I think 1 per query-type, e.g. 1 for offers (well, 2 as there is `offers:offer` and `offers:offer_deleted` to listen on, so not too many...

I thought it makes sense to move the logic to do on websocket update in the files where the queries take place, so they are together, but alternatively could centralized it (like we do now with the vuex commits).

The other idea I had is to only ever use the `id` coming from the websocket, so we can transition to a lighter "procotol" for websocket updates in the future, just based on ids.